aiida/orm/nodes/data/data.py

```diff
@@ -59,6 +59,7 @@ def clone(self):
 
         :returns: an unstored clone of this Data node
         """
+        # pylint: disable=no-member
```

Comment:
should maybe be more specific, on the exact line?

Reply:
done